src/m_vector.h

```diff
@@ -22,178 +22,186 @@
 
 #include "doomtype.h"
 
+#if defined(__clang__) && __has_attribute(ext_vector_type)
```

Comment:
Could you abstract this away and use the corresponding vector extension on gcc?

https://gcc.gnu.org/onlinedocs/gcc/Vector-Extensions.html

Reply:
No because gcc vector extension doesn't support xyzw components.
The only way to support it is to switch to array access operator[] for the existing struct vector types.

Reply:
I see, thank you for elaborating. @ceski-1 would you be so kind and check if this breaks anything on your end? I still don't have a gamepad with gyro and rumble, but Christmas is around the corner. :wink:

Reply:
Gyro works fine for me.